docs/integrations/sources/aws-cloudtrail.md

```diff
@@ -0,0 +1,48 @@
+# AWS CloudTrail
+
+## Overview
+
+The AWS CloudTrail source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+This Source Connector is based on a [Boto3 CloudTrail](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudtrail.html).
+
+### Output schema
+
+This Source is capable of syncing the following core Streams:
+
+* [Management Events](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudtrail.html#CloudTrail.Client.lookup_events)
+
+Insight events are not supported right now. Only Management events are available.
+
+### Data type mapping
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+| `string` | `string` |  |
+| `number` | `integer` |  |
+| `array` | `array` |  |
+| `object` | `object` |  |
+
+### Features
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental - Append Sync | Yes |  |
+| Namespaces | No |  |
+
+### Performance considerations
+
+The rate of lookup requests for `events` stream is limited to two per second, per account, per region. If this limit is exceeded, a throttling error occurs.
+
+## Getting started
+
+### Requirements
+
+* AWS Access key ID
+* AWS Secret access key
+* AWS region name
+
+### Setup guide
```

Comment:
could you add a section for the changelog like [here](https://github.com/airbytehq/airbyte/blob/master/docs/integrations/sources/stripe.md#changelog)

changelogs now live in the external facing docs